blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/ModelObjectSerializer.java

```diff
@@ -1,73 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright (c) 2016, CloudBees, Inc.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-package io.jenkins.blueocean.commons.stapler;
-
-import io.jenkins.blueocean.commons.stapler.export.ExportConfig;
-import io.jenkins.blueocean.commons.stapler.export.Flavor;
-import io.jenkins.blueocean.commons.stapler.export.Model;
-import io.jenkins.blueocean.commons.stapler.export.ModelBuilder;
-
-import javax.annotation.Nonnull;
-import java.io.IOException;
-import java.io.StringWriter;
-import java.io.Writer;
-
-/**
- * Simple Jenkins Model Object serializer.
- *
- * @author <a href="mailto:tom.fennelly@gmail.com">tom.fennelly@gmail.com</a>
- */
-public class ModelObjectSerializer {
```

Comment:
Since we are not using it, why not leave it as it is? guess it will make easier to  copy from stapler in future.